include/spdk/pci_ids.h

```diff
@@ -44,7 +44,9 @@ extern "C" {
 
 #include <stdint.h>
 
+#define SPDK_PCI_ANY_ID			0xffff
```

Comment:
Without the "+" sign in the patch, this line matches the indentation of other lines.